spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  before :each do
+    user = User.create(name: 'Akite donald', posts_counter: 0)
+    post = user.posts.create(title: 'Post 1', comments_counter: 0, likes_counter: 0)
+    user.save!
+    post.author_id = user.id
+    post.save!
+  end
+  describe 'GET /users/:id/posts' do
+    let(:user) { User.create(name: 'Akite donald', posts_counter: 0) }
+    let(:post) { Post.create(title: 'Post 1', comments_counter: 0, likes_counter: 0) }
+    it 'has a 200 status code' do
+      get '/', params: { user_id: user.id }
+      expect(response).to have_http_status(200)
+    end
+    it 'renders the right template for Posts' do
+      get '/', params: { user_id: user.id }
+      expect(response.body).to render_template(:index)
+    end
+  end
```

Comment:
- [ ] Here happens the same, please make sure to include a test for the `response.body` containing the correct placeholder text, just as you did in your `Post#show` tests! 🤓 🔧